src/mono/netcore/System.Private.CoreLib/src/System/Threading/WaitSubsystem.HandleManager.Unix.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Can you add these files to the shared partition, and ifdef them out for CoreCLR in the build script for now?

We will want to switch CoreCLR to this eventually as well, and having it in the shared place will makes it easier. It is similar how the portable threadpool was done. It was enabled for Mono only, and later for CoreCLR too.

Reply:
Will do.